src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -295,442 +295,507 @@ sylius_rbac:
+        sylius.manage.menu:
+            description: Manage menus
+            child_permissions: [sylius.menu.show, sylius.menu.index, sylius.menu.create, sylius.menu.update, sylius.menu.delete]
+        sylius.manage.route:
+            description: Manage routes
+            child_permissions: [sylius.route.show, sylius.route.index, sylius.route.create, sylius.route.update, sylius.route.delete]
+        sylius.manage.simple_block:
+            description: Manage simple blocks
+            child_permissions: [sylius.simple_block.show, sylius.simple_block.index, sylius.simple_block.create, sylius.simple_block.update, sylius.simple_block.delete]
+        sylius.manage.string_block:
+            description: Manage string blocks
+            child_permissions: [sylius.string_block.show, sylius.string_block.index, sylius.string_block.create, sylius.string_block.update, sylius.string_block.delete]
+        sylius.manage.imagine_block:
+            description: Manage imagine blocks
+            child_permissions: [sylius.imagine_block.show, sylius.imagine_block.index, sylius.imagine_block.create, sylius.imagine_block.update, sylius.imagine_block.delete]
+        sylius.manage.slideshow_block:
+            description: Manage slideshow blocks
+            child_permissions: [sylius.slideshow_block.show, sylius.slideshow_block.index, sylius.slideshow_block.create, sylius.slideshow_block.update, sylius.slideshow_block.delete]
+        sylius.manage.slideshow:
+            description: Manage slideshows
+            child_permissions: [sylius.slideshow.show, sylius.slideshow.index, sylius.slideshow.create, sylius.slideshow.update, sylius.slideshow.delete]
+        sylius.manage.email:
+            description: Manage emails
+            child_permissions: [sylius.email.show, sylius.email.index, sylius.email.create, sylius.email.update, sylius.email.delete]
+        sylius.manage.report:
+            description: Manage reports
+            child_permissions: [sylius.report.show, sylius.report.index, sylius.report.create, sylius.report.update, sylius.report.delete]
+        sylius.manage.contact_request:
+            description: Manage contact requests
+            child_permissions: [sylius.contact_request.show, sylius.contact_request.index, sylius.contact_request.create, sylius.contact_request.update, sylius.contact_request.delete]
+        sylius.manage.contact_topic:
+            description: Manage contact topics
+            child_permissions: [sylius.contact_topic.show, sylius.contact_topic.index, sylius.contact_topic.create, sylius.contact_topic.update, sylius.contact_topic.delete]
+        sylius.manage.settings:
+            description: Manage settingss
+            child_permissions: [sylius.settings.general, sylius.settings.security, sylius.settings.taxation]
+
+        sylius.catalog:
```

Comment:
Should these have the 'manage' prefix as well? or some other prefix if they're the level above?